packages/playwright-core/src/utils/happy-eyeballs.ts

```diff
@@ -45,8 +45,9 @@ class HttpsHappyEyeballsAgent extends https.Agent {
   }
 }
 
-export const httpsHappyEyeballsAgent = new HttpsHappyEyeballsAgent();
-export const httpHappyEyeballsAgent = new HttpHappyEyeballsAgent();
+// These options are aligned with the default Node.js globalAgent options.
+export const httpsHappyEyeballsAgent = new HttpsHappyEyeballsAgent({ keepAlive: true, scheduling: 'lifo', timeout: 5000 });
```

Comment:
scheduling: 'lifo' - surprised to see it lifo
timeout: 5000 - concerned that it might bite us, what is the present value?